py3status/modules/google_calendar.py

```diff
@@ -317,6 +339,9 @@ def _build_response(self):
         index = 0
 
         for event in self.events:
+            self.py3.threshold_get_color(index + 1, 'event')
+            self.py3.threshold_get_color(index + 1, 'time')
```

Comment:
Duplicate color thresholds?

Reply:
No, the `event` threshold corresponds to the event names. The `time` thresholds are for the start/end times and the date. So we can have different colors for them. Without the second line, only the event colors will be applied.